cnd/src/respawn.rs

```diff
@@ -48,6 +48,57 @@ pub async fn respawn(storage: Storage, protocol_spawner: ProtocolSpawner) -> any
                 protocol_spawner.spawn(id, swap.alpha, Utc::now().naive_local(), Side::Alpha, role);
                 protocol_spawner.spawn(id, swap.beta, Utc::now().naive_local(), Side::Beta, role);
             }
+            Swap {
+                id,
+                role,
+                alpha: Protocol::Halight,
+                beta: Protocol::Herc20,
+            } => {
+                let swap: Swap<halight::Params, herc20::Params> = match storage.load(id).await {
+                    Ok(swap) => swap,
+                    Err(e) => {
+                        tracing::warn!("failed to load data for swap {}: {:?}", id, e);
+                        continue;
+                    }
+                };
+
+                protocol_spawner.spawn(id, swap.alpha, Utc::now().naive_local(), Side::Alpha, role);
```

Comment:
We need a ticket for fixing the `start_of_swap` timestamp.

Reply:
Can you elaborate? What's the desired behaviour?

Reply:
I guess this is resolved with #2790.